src/h5web/dataset-visualizer/DimensionMapper.module.css

```diff
@@ -0,0 +1,87 @@
+.mapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: var(--primary-light-bg);
+}
+
+.buttonGroupWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  margin: 0.25rem;
+}
+
+.buttonGroup {
+  display: flex;
+}
+
+.buttonGroupLabel {
+  margin-right: 0.5rem;
+}
+
+.btn {
+  composes: btn-clean from global;
+  display: flex;
+  align-items: center;
+  padding: 0.25rem 0.75rem 0.375rem;
+  background-color: var(--secondary);
+  transition: background-color 0s ease-out;
+  color: white;
+  border-radius: 10%;
+}
+
+.btn:hover {
+  background-color: var(--secondary-dark);
+}
+
+.btn[aria-checked='true'],
+.btn[aria-pressed='true'] {
+  background-color: var(--secondary-dark);
+  font-weight: bold;
+}
+
+.sliderWrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 1rem 0;
+}
+
+.slider {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-self: center;
+  margin: 0.5rem 0.125rem;
+  height: 10rem;
+  min-width: 1.75rem;
+}
+
+.sliderTrack {
+  width: 2px;
+  background-color: var(--secondary-dark);
+}
+
+.sliderButton {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0.125rem;
+  background-color: var(--primary-bg);
+  border: 2px solid var(--secondary);
+  border-radius: 1.5em;
+}
+
+.sliderThumb {
+  composes: sliderButton;
+  font-size: 0.625rem;
+  height: 1.5rem;
+  min-width: 1.5rem;
+  cursor: grab;
+}
+
+.sliderThumb:global(.active) {
+  cursor: none;
+  border-color: var(--secondary-dark);
+}
```

Comment:
The CSS was fetched from other components modules and adapted. I hope to do something cleaner once the toolbar refactor is finished.